lib/components/inputs/UploadField/UploadField.tsx

```diff
@@ -1,10 +1,11 @@
-import React, { useId, useState } from 'react'
+import React, { ReactNode, useId, useState } from 'react'
 import clsx from 'clsx'
 import { IconButton } from '@mui/material'
 import SpanTooltip from '../../SpanTooltip'
 import { EMPTY_STRING, NOP } from 'consts'
 import { Close } from 'svgs'
 import Tooltip from '../../Tooltip'
+import Info from '../../Info'
 
 import './uploadField.scss'
 const ENCODING_TYPES = { text: 'text', binary: 'binary', base64: 'base64' }
```

Comment:
I suggest to move this to a const, so that it's used in other projects by that const (like we have with `TOASTER_TYPES`)
and replace 'binary' in wekapp

Reply:
I prefer capital letters for constants 
```
ENCODING_TYPES = { TEXT: 'text', BINARY: 'binary', BASE64: 'base64' }